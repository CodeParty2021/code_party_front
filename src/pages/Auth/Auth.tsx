import React from "react";
import { firebaseConfig } from "../../config";
type Prop = {};

export const Auth: React.FC<Prop> = () => {
  console.log(firebaseConfig);
  return (
    <div>
      <h1>Auth</h1>
    </div>
  );
};
