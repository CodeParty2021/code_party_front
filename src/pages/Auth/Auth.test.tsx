import React from "react";
import { shallow } from "enzyme";
import { Auth } from "./Auth";
describe("<Auth />", () => {
  it("auth snapshot test", () => {
    const wrapper = shallow(<Auth />);

    expect(wrapper.getElements()).toMatchSnapshot();
  });
});
