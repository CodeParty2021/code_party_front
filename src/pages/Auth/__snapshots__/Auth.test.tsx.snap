// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Auth /> auth snapshot test 1`] = `
Array [
  <div>
    <div>
      <h1>
        Auth
      </h1>
    </div>
    <div>
      <SignInScreen />
    </div>
  </div>,
]
`;
